src/sidebar/util/annotation-sharing.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const serviceConfig = require('../service-config');
+
+/**
+ * Is the sharing of annotations enabled? Check for any defined `serviceConfig`,
+ * but default to `true` if none found.
+ *
+ * @param {object} settings
+ * @return {boolean}
+ */
+function sharingEnabled(settings) {
+  const serviceConfig_ = serviceConfig(settings);
+  if (serviceConfig_ === null) {
+    return true;
+  }
+  if (typeof serviceConfig_.enableShareLinks !== 'boolean') {
+    return true;
+  }
+  return serviceConfig_.enableShareLinks;
+}
+
+/**
+ * Return any defined standalone URI for this `annotation`, preferably the
+ * `incontext` URI, but fallback to `html` link if not present.
+ *
+ * @param {object} annotation
```

Comment:
This reminds me, I think we ought to (not in this PR) introduce a centralized definition for types of objects from the API so that we can use a more specific type (`Annotation`, `Group` etc.) rather than just use `object`, which isn't helpful for the reader or static analysis.